src/components/ProjectCard.tsx

```diff
@@ -2,9 +2,11 @@ import React, { FC } from "react";
 import { Project, ExperienceLevelsTypes } from "../types";
 import "../style/components/ProjectCard.css";
 
-
 const getTags = (tagList: Array<string>) => {
-  return tagList.map(tag => {
+  if (tagList.length > 5) {
```

Comment:
Since #118 is still in progress, do you mind adding some mock data and taking a screenshot, so we can see how it would look like with the tags on? Feel free to add it to the PR description.

Reply:
Sure I just updated the PR description with a screenshot. I asked during the open source zoom call if the CSS needed to be changed for the tags, and I was told it was not needed. So I have no changed the styling of the tags.